### ISSUE TITLE
Codevoorbeelden uit formulierrichtlijnen hebben onleesbare layout

### ISSUE THREAD

**Issue by Robbert** [nl-design-system/documentatie#3719]
## User Story

Als gebruiker van de richtlijnen wil ik graag dat de voorbeelden bij de richtlijnen op https://nldesignsystem.nl/richtlijnen er goed uit zien, zodat ik een duidelijk voorbeeld heb om de richtlijn beter mee te begrijpen.

## Beschrijving

<img width="716" height="374" alt="Image" src="https://github.com/user-attachments/assets/c9e1b184-dc8f-4511-9bce-992f1db294e5" />

https://nldesignsystem.nl/richtlijnen/formulieren/voorkom-fouten/verplichte-velden/

Gemeld door Robert Roose in Slack: https://codefornl.slack.com/archives/C7KDQRXMZ/p1770800505625959

Doordat de codevoorbeelden van de formulierrichtlijnen zeer minimalistisch zijn gemaakt, is de resulterende layout onvoldoende leesbaar.

Bijvoorbeeld: veel voorbeelden hebben alleen een `<label>` en een `<input>` element. Daardoor staan de teksten op 1 regel, tenzij er speciale CSS wordt toegevoegd om `label { display: block }` te doen.

Het zou beter zijn als de HTML zonder extra CSS goed werkt.

## Acceptatiecriteria

- [ ] Alle formulierrichtlijnen zien er ook goed uit zonder CSS
